docs/Alpine.md

```diff
@@ -23,3 +23,30 @@ class Explanation extends \Magewirephp\Magewire\Component
 > ```$wire``` gives you access to the Magewire component properties using AlpineJS. To avoid a misunderstanding your could
 > also use ```wire:click="$set('foo', Math.random())'"``` to set the ```foo``` property value. It just to demonstrate
 > how ```x-text``` could be used in combination with Magewire.
+
+### Accessing Component Array Properties
+```php
+class Explanation extends \Magewirephp\Magewire\Component
+{
+    public $foo = [
+        0 => [    
+            'foo' => 'bar',
+        ],
+        'nonNumericKey' => [
+            0 => 'fooBar',
+        ],
+    ];
+}
+```
+
+```html
+<div>
+    <div x-data>
+        <!-- Output 'bar' -->
+        <span x-text="$wire.foo[0]['foo']"></span>
+        
+        <!-- Output 'fooBar' -->
+        <span x-text="$wire.foo['nonNumericKey'][0]"></span>
```

Comment:
@pykettk, did you also try $wire.foo.nonNumericKey['0']? Would love to see that in there as an example when that does work.

Reply:
@wpoortman I didn't - I'll give that a try later on and update here as required, good idea 👍 Thanks 🙂

Reply:
I think that will work also 😅

Reply:
@wpoortman It does - I just tested to confirm 😅 I've added a couple more examples so they should now cover all bases 🤞